apps/cli/commands/ingest_docker_compose.py

```
@@ -0,0 +1,224 @@
+    try:
+        # Load config
+        config = get_config()
+
+        # Display starting message
+        console.print(f"[yellow]Starting ingestion: {file_path}[/yellow]")
+
+        # Track start time for duration calculation
+        start_time = datetime.now(UTC)
+
+        # Create reader and load data
+        logger.info(f"Parsing docker-compose file: {file_path}")
```

Ruff (0.14.1)

Logging statement uses f-string

(G004)